docs/theming.md

```diff
@@ -109,15 +109,91 @@ dark theme.
 In order to style your own components with our tooling, the component's styles must be defined 
 with Sass. 
 
-You can consume the theming functions and variables from the `@angular/material/core/theming`.
```

Comment:
Rather than making this document much longer, I'd rather create another file `theming-your-components.md` that covers how to theme your own components with the material tools.